Codecademy/gamut
# feat(Illustration): add LightBulbGears illustration

### Overview


- add LightBulbGears illustration to gamut-illustrations


### PR Checklist

- [x] Related to designs: https://www.figma.com/file/RXTfBM4wAe53JfYeKPwxje/Code-Challenges-2.0?type=design&node-id=2251-7361&t=hMmDbhPuHHgQFykU-0
- [x] Related to JIRA ticket: [LEGO-3344]

#### Testing Instructions
1. Visit Storybook and [go to Illustrations](https://646e344a362fb80a612df0f4--gamut-preview.netlify.app/)
2. See new LightBulbGears illustrations.

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |




[LEGO-3344]: https://codecademy.atlassian.net/browse/LEGO-3344?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
